stubs/flake8-2020/@tests/stubtest_allowlist.txt

```diff
@@ -0,0 +1,5 @@
+# We don't need violation definitions:
+flake8_2020.YTT\d+
+
+# Has many implementation details:
+flake8_2020.Visitor
```

Comment:
I don't think this actually has any public methods that aren't already included in typeshed's stub for the base class, `ast.NodeVisitor`: https://github.com/python/typeshed/blob/38d0f691e6afba8e88347726b7d56ed361ffd7bf/stdlib/ast.pyi#L37